whitepaper/chapter01.md

```diff
@@ -0,0 +1,140 @@
+# 1. Cloud Native Thinking for Telecommunications
+
+Basic tenets and end users suggestions on how cloud native design and principles can be applied to mission critical telecommunications functions.
+
+## Table of Contents
+* [1.1 Introduction](#1.1)
+* [1.2 What is Cloud Native?](#1.2)
+* [1.3 Defining Cloud Native Systems](#1.3)
+  * [1.3.1 What "loosely coupled" means in cloud native systems](#1.3.1)
+  * [1.3.2 Cloud native applications require orchestration](#1.3.2)
+  * [1.3.3 Infrastructure, Deployment and Configuration of Cloud Native Systems](#1.3.3)
+* [1.4 Cloud Native Network Functions](#1.4)
+* [1.5 Cloud Native for Telcos](#1.5)
+  * [1.5.1 A Brief History of Virtualisation for Telcos](#1.5.1)
+  * [1.5.2 Software Defined Networking And The Emergence of VNFs](#1.5.2)
+  * [1.5.3 Principles for Cloud Native Telco Infrastructure and Applications](#1.5.3)
+  * [1.5.4 Evolving the Stack From VNFs to CNFs](#1.5.4)
+* [1.6 Cloud Native for Telcos in Practice](#1.6)
+* [1.7 Conclusion](#1.7)
+
+<a name="1.1"></a>
+## 1.1 Introduction
+
+There has been a steady increase in the adoption of cloud native technologies to build infrastructure and run workloads across the board, to the point that it's now the defacto choice for enterprise and IT. It makes perfect sense that the Telecom industry wants to benefit in a similar way with it's platforms, networks and telephony applications.
```

Comment:
I would rephrase to "It then makes perfect sense for the Telecom industry to follow in order to benefit in a similar way with it's platforms ..."